rviz_rendering/include/rviz_rendering/point_cloud.hpp

```diff
@@ -27,8 +27,14 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef OGRE_TOOLS_OGRE_POINT_CLOUD_H
-#define OGRE_TOOLS_OGRE_POINT_CLOUD_H
+#ifndef RVIZ_RENDERING__POINT_CLOUD_HPP_
+#define RVIZ_RENDERING__POINT_CLOUD_HPP_
+
+#include <stdint.h>
```

Comment:
Use `cstdint` instead.

Reply:
Will be addressed in the refactoring PR.